app/src/main/java/net/gsantner/markor/format/TextConverter.java

```diff
@@ -84,7 +84,7 @@ public String convertMarkupShowInWebView(Document document, String content, WebV
         }
 
         String baseFolder = new AppSettings(context).getNotebookDirectoryAsStr();
-        if (document.getFile() != null && document.getFile().getParentFile() != null) {
```

Comment:
document requires file != null. If file == null, construction of document will fail immediately. Therefore these checks are redundant and misleading. Removed document file null checks